### PR TITLE
Add Logistic Moment

### PR DESCRIPTION
**Pull request by shivam15s**
Add moment and tests for the below distribution as part of #5078 
+  pymc.distributions.continuous.Logistic

### CODE REVIEW

File: pymc/tests/test_distributions_moments.py [hunk 1]
```diff
@@ -341,3 +342,22 @@ def test_weibull_moment(alpha, beta, size, expected):
     with Model() as model:
         Weibull("x", alpha=alpha, beta=beta, size=size)
     assert_moment_is_expected(model, expected)
+
+@pytest.mark.parametrize(
+    "mu, s, size, expected",
+    [
+        (1, 1, None, 1),
+        (1, 1, 5, np.full(5, 1)),
+        (np.arange(1, 6), np.arange(1, 6), None, np.arange(1, 6)),
```
Comment (ricardoV94) [inline pymc/tests/test_distributions_moments.py]: One of the test conditions should be changed to have a vector `s` and scalar `mu`, to make sure the `broadcast` is being respected with respect to the parameter that does not matter `s`. For instance:

```suggestion
        (2, np.arange(1, 6), None, np.full(5, 2)),
```

**Review by ricardoV94 — CHANGES_REQUESTED** [review]
Small suggestion to make the test more robust.

Also thanks for opening a PR!

**Comment by ricardoV94** [comment]
@shivam15s There are some conflicts that have to be resolved due to other PRs having been merged in the meantime

**Review by ricardoV94 — APPROVED** [review]

**Comment by shivam15s** [comment]
@ricardoV94 Don't know why `test_bart.py` failed. It seems to work locally.
image

**Comment by ricardoV94** [comment]
Seems related to https://github.com/aesara-devs/aeppl/issues/84

Should not be affected by this PR

**Comment by ricardoV94** [comment]
Unfortunately you will have to resolve conflicts once again. There were more PRs that were just merged.

**Comment by ricardoV94** [comment]
Just merged. Thanks a lot @shivam15s and looking forward to your next PR

**Pull request merged**